metpy/plots/_util.py

```diff
@@ -6,6 +6,7 @@
 from datetime import datetime
 import posixpath
 
+from matplotlib import patheffects
```

Comment:
We usually do this as:
```python
import matplotlib.patheffects as mpatheffects
```

Reply:
Will change here, but in that case should probably change all the workshop material as well as it all uses the current method: https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Satellite_Data/Working%20with%20Satellite%20Data.ipynb